projects/cdk/directives/auto-focus/autofocus.directive.ts

```diff
@@ -84,12 +86,33 @@ export class TuiAutoFocusDirective implements AfterViewInit {
         return this.element.matches('input, textarea');
     }
 
+    private isFullScreenDialog() {
```

Comment:
This only takes into account default built-in implementation of dialogs and relies on its "private" implementation of `data-size` attribute. That's not a reliable solution and it doesn't cover all use cases.

Reply:
removed it